app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalAuthenticateController.php

```diff
@@ -77,6 +80,10 @@ public function __invoke(KnowledgeManagementPortalAuthenticateRequest $request,
             'success' => true,
             'token' => $token->plainTextToken,
             'user' => auth('contact')->user(),
+            'service_management_enabled' => $settings->knowledge_management_portal_service_management,
+            'has_assets' => auth()->guard('contact')->check() ? auth()->guard('contact')->user()->assetCheckIns()->exists() || auth()->guard('contact')->user()->assetCheckOuts()->exists() : false,
+            'has_license' => auth()->guard('contact')->check() ? auth()->guard('contact')->user()->productLicenses()->exists() : false,
+            'has_tasks' => auth()->guard('contact')->check() ? auth()->guard('contact')->user()->tasks()->exists() : false,
```

Comment:
Is it possible to simplify these checks to be like:

```php
 auth()->guard('contact')->user()?->productLicenses()->exists()
```

That way we can get rid of the `auth()->guard('contact')->check()` in each?